src/components/select/select.js

```diff
@@ -288,19 +288,16 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
             }
           });
 
-        // Wait until postDigest so that we attach after ngModel's
-        // blur listener so we can set untouched.
-        $mdUtil.nextTick(function () {
-          element.on('blur', function() {
-            if (untouched) {
-              untouched = false;
-              ngModelCtrl.$setUntouched();
-            }
 
-            if (selectScope.isOpen) return;
-            containerCtrl && containerCtrl.setFocused(false);
-            inputCheckValue();
-          });
+        element.on('blur', function() {
```

Comment:
Commenting out that whole handler makes these fail:
* `<md-select> should set touched only after closing`.
*  ` <md-select> should remove the input-container focus state FAILED`.

As for the event not firing, my point is that the disabled attribute can be removed at a later point and it'll need to fire then.